src/common/MessageProcessor.sol

```diff
@@ -247,15 +237,13 @@ contract MessageProcessor is Auth, IMessageProcessor {
                 m.isIncrease
             );
         } else if (kind == MessageType.UpdateShares) {
+            // TODO: Remove price from hub
```

Comment:
Note: open TODO here

Reply:
@onnovisser can be fixed with current main